lerna.json

```diff
@@ -6,6 +6,11 @@
     "packages/@aws-cdk/*",
     "tools/*"
   ],
+  "command": {
+    "bootstrap": {
+      "npmClientArgs": ["--no-package-lock"]
```

Comment:
why?

Reply:
Getting constant complaints by contributors about their `package-locks` being updated, which are not part of their commit and they have to manually revert those changes.